carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -281,7 +281,7 @@ void WMBroadcaster::geofenceFromMsg(std::shared_ptr<Geofence> gf_ptr, const cav_
   if (msg_detail.choice == cav_msgs::TrafficControlDetail::MAXSPEED_CHOICE) 
   {  
     //Acquire speed limit information from TafficControlDetail msg
-    sL = lanelet::Velocity(msg_detail.maxspeed * lanelet::units::MPH()); 
+    sL = lanelet::Velocity(msg_detail.maxspeed * MS_TO_MPH * lanelet::units::MPH()); 
```

Comment:
Rather than adding this multiplication you could have just changed the units I believe

```
lanelet::Velocity(msg_detail.maxspeed * lanelet::units::MPS()); 
```


Same below. Add you could remove the extra constant

Reply:
Fixed.